tests/generate.py

```diff
@@ -664,13 +668,21 @@ def _cert_path(cert_type: str) -> str:
             issuer_name=issuer_subject,
         )
 
+        write_der(
+            _rpk_path(name),
+            certificate.public_key().public_bytes(
+                Encoding.DER, PublicFormat.SubjectPublicKeyInfo
+            ),
+            force,
+        )
```

Comment:
Currently the tests fail, hopefully because I'm incorrectly writing the public key DER. Does any of you know what the arguments should be for `public_bytes` to write the public key.
I've also tried `private_key.public_key().public_bytes(..., ...)`, but in that case as well the key_value in `signed_data:verify_signature`  for `ee.der` and `spki.der` are different.